201-vm-linux-comprehensive-with-managed-disks/azuredeploy.json

```diff
@@ -237,7 +238,7 @@
         "typeHandlerVersion": "2.0",
         "autoUpgradeMinorVersion": "true",
         "settings": {
-          "fileUris": ["[concat(parameters('_artifactsLocation'), variables('scriptName'), '.sh')]"],
+          "fileUris": [ "[concat(parameters('_artifactsLocation'), variables('scriptName'), '.sh')]" ],
```

Comment:
whats the change here..

Reply:
Ah just whitespaces..

Reply:
Visual Studio fixes the spaces when you save, that's all.